src/router.ts

```diff
@@ -24,8 +24,9 @@ const router = createRouter({
       path: "/logout-callback",
       name: "logout-callback",
       beforeEnter(to) {
-        document.location = `/api/france-connect${to.href}`
+        document.location = `/api/france-connect${to.fullPath}`
```

Comment:
to.href n'existe pas mais ça reste à tester

Reply:
Cela renvoie la même chose, mais href n'est pas recommandé ici, on préférera utiliser `to.fullPath`